### ISSUE TITLE
Suggestions for talks

### ISSUE THREAD

**Issue by overlookmotel** [lnug/feedback#97]
Hi all.

Attended LNUG for the first time tonight (in fact first time I've attended any code-related meet-up). Had a great time - interesting talks, interesting people and a friendly atmosphere. And, yeah, free pizza... Thanks very much for organising.

You were soliciting the involvement of newbies, so... here I am...

I don't know how much use this is, but as a relative newbie to node, there's a couple of topics which are a consistent source of frustration for me, and which I'd love to hear someone more knowledgeable than I speak about at a future LNUG:

#### 1. NPM: How to sift the wheat from the chaff

Obviously the NPM ecosystem is amazing and there's probably a module out there for pretty much anything you want to do. But finding them is a recurring problem, and I'm convinced that half the modules I've published have probably already been done elsewhere, but it's difficult to find them.

What tools are there for more effectively searching NPM for what you're looking for? How to identify which existing modules are reliable and well-maintained? And, if good NPM search doesn't yet exist, how could we go about building it?

#### 2. NoSQL: how to get the advantages of NoSQL without losing SQL's benefits

As a developer who grew up in the world of MySQL and Postgres, I hear all the talk of NoSQL databases but haven't yet been able to make the leap.

Time and time again, I've looked at Mongo, CouchDB etc. I like the way they're less monolithic and all the pub/sub stuff would be really useful, but can't get my head around how to switch without losing the central advantages of old-skool SQL. Strangely I've found little writing on the web that speaks to this point.

How, for example, can you produce atomic updates and transactions? Or how do you achieve the same things by another means with NoSQL?

Maybe these topics are a bit basic, but just thought I'd throw in my two cents.

Thanks again for a great night.

**Comment by iancrowther** [comment]
This is great feedback, ty.

I'm glad you had a good time 😀

We'll be sure to add it into the mix at the next meeting

**Comment by orliesaurus** [comment]
1. You usually google them if you don't know them by ❤️  - StackOverflow answers with hundreds of votes are usually trustworthy
2. Great question, my usual answer is that unless you fall into that specific category you're probably better off using something like PostgreSQL. However I hope someone submits a talk about this particular issue of choosing the right db for the job

**Comment by overlookmotel** [comment]
@admataz Yes, I'd still be very much interested in doing a talk. But work has completely taken over my life last few months. When I've got my current project out the door and had time to breathe, I'll revive the thread relating to that.
